pkg/registry/rbac/rolebinding/policybased/storage.go

```diff
@@ -46,6 +47,14 @@ func (s *Storage) Create(ctx api.Context, obj runtime.Object) (runtime.Object, e
 		if s.superUser != "" && user.GetName() == s.superUser {
 			return s.StandardStorage.Create(ctx, obj)
 		}
+
+		// system:masters is special because the API server uses it for privileged loopback connections
```

Comment:
@ericchiang This is probably the most controversial change.  Remember that the loopback authorization we use actually inspects to match on the known group name.  This means that we know that `system:masters` group is never escalating its permissions when creating policy resources.  This is useful for allowing bootstrapping to actually use the API server instead of going direct to etcd.  It's always allowed to add roles, even when no roles exist ahead of time.